web_src/less/_user.less

```diff
@@ -5,8 +5,16 @@
 
     &.profile {
         .ui.card {
-            .username {
-                display: block;
+            .header {
+                font-weight: 700;
+                font-size: 1.3em;
+                margin-top: -0.2em;
+                line-height: 1.3em;
```

Comment:
Maybe use `rem` units? They should be preferred unless you really want inheritance.

Reply:
Done.